sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -685,4 +685,17 @@ private[speedy] case class PartialTransaction(
     go(this)
   }
 
+  def transactionTrace: List[(NodeId, Node.Exercise)] = {
+    def go(context: Context): List[(NodeId, Node.Exercise)] =
```

Comment:
Is there any chance this may stack overflow or will something go wrong anyway if the context is that big?

Reply:
Let's go with an iterator :trollface: